files/dush/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "dush",
+  "repo": "tunnckoCore/dush",
+  "files": {
+    "include": ["dist/dush.js", "dist/dush.min.js", "dist/dush.standalone.js", "dist/dush.standalone.min.js"]
```

Comment:
angular.moment is there because of legacy reasons, the first version had it before we started pushing for min only https://github.com/jsdelivr/jsdelivr/tree/master/files/angular.moment/0.10.0

Are you 100% sure the non-min files are needed on the CDN? I can add them but only if they are really necessary